packages/interfaces/src/index.ts

```diff
@@ -11,6 +11,7 @@ export enum Protocols {
   Store = "store",
   LightPush = "lightpush",
   Filter = "filter",
+  PeerExchange = "peer-exchange",
```

Comment:
```suggestion
  PeerExchange = "peerexchange",
```

Let's keep same casing convention than light push (or you can change light push instead but be sure to update the changelog to note the breaking change).

Reply:
The difference is Peer Exchange is defined as [`peer-exchange` in the protocol identifier in the specs],(https://rfc.vac.dev/spec/34/) and [Light Push as `lightpush`](https://rfc.vac.dev/spec/19/) -- so keeping it consistent with them?